src/reactable.jsx

```diff
@@ -344,6 +344,7 @@
             var Ths = [];
             for (var index = 0; index < this.props.columns.length; index++) {
                 var column = this.props.columns[index];
+                var thClass = column.key
```

Comment:
let's go with `.replace(/\s+/g, '-')` to catch other non-space whitespace characters as well.